helm_resource/helm-delete-helper.py

```diff
@@ -19,6 +21,26 @@
   stderr=subprocess.DEVNULL,
 )
 if status_result == 0:
+    # Extract CRDs from managed manifests
+    get_manifest_cmd = ('helm', 'get', 'manifest') + namespace_args + (release_name,)
+    print('Running cmd: %s' % ' '.join(get_manifest_cmd), file=sys.stderr)
+    raw_managed_resources = subprocess.check_output(get_manifest_cmd)
+    managed_resources = list(yaml.load_all(resources, Loader=yaml.SafeLoader))
```

Comment:
pyyaml is not available. An alternative may be using `grep -Poz '(?s)\s*kind: CustomResourceDefinition.*?metadata:.*?\s+name: \K([-\.a-z0-9]*\n)'`